### PR TITLE
feat(db): add name column to movies table

### PR DESCRIPTION
**Pull request by marklee22**
**What: **Adding the name column to the movies db table.

Diff based off of: [feat(movies): Adding POST endpoint + unit tests](https://github.com/marklee22/sfmovies/pull/7)

**Why:** Following guide.

screen shot 2017-09-15 at 1 54 37 pm

### CODE REVIEW

**Review by sjl2 — COMMENTED** [review]
Since this adds a column constraint, let's have this migration not be [run in a migration](https://www.notion.so/lob/Writing-Migrations-15caf9dd14f74c47b498fd7bee8a7b51#d3867026f2a349cc81a13c2bbf6a4c22).

**Review by sjl2 — APPROVED** [review]
LGTM upon removal of transaction!

File: db/migrations/20170915134724_add_name_to_movies.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+exports.config = { transaction: false };
```
Comment (kienan) [inline db/migrations/20170915134724_add_name_to_movies.js]: Can you move this to the bottom of the file?

**Review by kienan — APPROVED** [review]
LGTM after moving the `exports.config` line!

**Pull request merged**